cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -626,4 +630,210 @@ define([
+            verifyEditingGroup(view, true, newGroupIndex);
+            saveOrCancel(view, options, newGroupIndex);
+        };
+
+        editExistingGroup = function(view, options) {
+            var groupIndex = options.groupIndex || 0;
+            view.$(scopedGroupSelector(groupIndex, '.edit')).click();
+            view.$(scopedGroupSelector(groupIndex, inputCss)).val(options.newName);
+            saveOrCancel(view, options, groupIndex);
+        };
+
+        verifyEditingGroup = function(view, expectEditing, index) {
+            // Should prevent the user from opening more than one edit
+            // form at a time by removing the add button(s) when
+            // editing a group.
+            index = index || 0;
+            if (expectEditing) {
+                expect(view.$(scopedGroupSelector(index, '.group-configuration-edit'))).toExist();
+                expect(view.$(newGroupCss)).not.toExist();
+                expect(view.$(addGroupCss)).not.toExist();
+            } else {
+                expect(view.$('.group-configuration-edit')).not.toExist();
+                if (view.collection.length === 0) {
+                    expect(view.$(newGroupCss)).toExist();
+                    expect(view.$(addGroupCss)).not.toExist();
+                } else {
+                    expect(view.$(newGroupCss)).not.toExist();
+                    expect(view.$(addGroupCss)).toExist();
+                }
+            }
+        };
+
+        respondToSave = function(requests, view) {
+            expect(requests.length).toBe(1);
+            expect(requests[0].method).toBe('POST');
+            expect(requests[0].url).toBe('/mock_url/0');
+            AjaxHelpers.respondWithJson(requests, {
+                name: 'Group Configuration',
+                groups: view.collection.map(function(groupModel, index) {
+                    return _.extend(groupModel.toJSON(), {id: index});
+                })
+            });
+        };
+
+        correctValidationError = function(view, requests, newGroupName) {
+            expect(view.$(validationErrorCss)).toExist();
+            verifyEditingGroup(view, true);
+            view.$(inputCss).val(newGroupName);
+            view.$(saveButtonCss).click();
+            respondToSave(requests, view);
+            expect(view.$(validationErrorCss)).not.toExist();
+        };
+
+        expectGroupsVisible = function(view, groupNames) {
+            _.each(groupNames, function(groupName) {
+                expect(view.$('.group-configurations-list-item')).toContainText(groupName);
+            });
+        };
+
+        beforeEach(function() {
+            TemplateHelpers.installTemplates(
+                ['no-content-groups', 'content-group-editor', 'content-group-details']
+            );
+        });
+
+        it('shows a message when no groups are present', function() {
+            expect(renderView().$('.no-group-configurations-content')).toExist();
```

Comment:
Check the actual message? The experiment group configuration test does, so it seems sort of odd that this one does not (also given the description of the test).